### ISSUE TITLE
Autoscroll on drag'n'drop when putting items inside is disabled works bad

### ISSUE THREAD

**Issue by Clans** [mbraak/jqTree#626]
Hello, 
I've disabled the ability to put nodes inside other nodes using this code:
```
onCanMoveTo: function(moved_node, target_node, position) {
         return position != 'inside';
}
```
and when I'm dragging the node to the very bottom the tree scrolls a bit and then stops. To make it scroll down a bit more I have to drag the node a bit up and then down again without releasing it.
Without this code, autoscroll works well.

**Comment by mbraak** [comment]
I'll have a look.

**Comment by mbraak** [comment]
I tried this with the devserver example, but I couldn't reproduce the error. I'll have a look in the code to see if I can find a clue.

**Comment by Clans** [comment]
I assume this may be because the area of the node itself is skipped while dragging and the code responsible for scrolling counts only the area between the nodes. In this case, there's not enough space to calculate the autoscroll when the drag has passed the position between the nodes but haven't reached the one yet. In other words, it may not work when the drag position is right in the middle of the node which is being skipped in my case.
Hope this will help somehow.

**Comment by mbraak** [comment]
I added a pr with a possible fix: #627. This is in theory an improvement of the autoscroll, but I couldn't reproduce it myself.

**Comment by Clans** [comment]
With this fix it works much better. Thank you! The library is awesome.
